server/src/main/java/com/cloud/consoleproxy/AgentHookBase.java

```diff
@@ -213,7 +213,7 @@ public void startAgentHttpHandlerInVM(StartupProxyCommand startupCmd) {
             byte[] ksBits = null;
 
             String consoleProxyUrlDomain = _configDao.getValue(Config.ConsoleProxyUrlDomain.key());
-            String consoleProxySslEnabled = _configDao.getValue("consoleproxy.sslEnabled");
+            String consoleProxySslEnabled = _configDao.getValue(ConsoleProxyManager.ConsoleProxySslEnabled.key());
```

Comment:
```suggestion
            String consoleProxySslEnabled = ConsoleProxyManager.ConsoleProxySslEnabled.value();
```

Reply:
@GutoVeronezi 
this may not work as the `_configDao` is set in the constructor of the class.
```
    protected AgentHookBase(VMInstanceDao instanceDao, HostDao hostDao, ConfigurationDao cfgDao, KeystoreManager ksMgr,
                         AgentManager agentMgr, KeysManager keysMgr, ConsoleAccessManager consoleAccessMgr) {
        _instanceDao = instanceDao;
        _hostDao = hostDao;
        _agentMgr = agentMgr;
        _configDao = cfgDao;
        _ksMgr = ksMgr;
        _keysMgr = keysMgr;
        consoleAccessManager = consoleAccessMgr;
    }
```

Reply:
If the class will have access to the DB, it should work, as the `value()` will query for the value.

Reply:
I did some debugging, it looks the values are the same.
```
2024-03-21T08:05:48,498 DEBUG [c.c.c.ConsoleProxyManagerImpl] (main:[]) (logid:) configs.get(ConsoleProxySslEnabled.key() = true
2024-03-21T08:05:48,502 DEBUG [c.c.c.ConsoleProxyManagerImpl] (main:[]) (logid:) ConsoleProxySslEnabled.value() = true

2024-03-21T08:06:02,499 DEBUG [c.c.c.C.VmBasedAgentHook] (AgentConnectTaskPool-4:[ctx-f64d74cc]) (logid:29169184) _configDao.getValue(ConsoleProxyManager.ConsoleProxySslEnabled.key()) = true
2024-03-21T08:06:02,499 DEBUG [c.c.c.C.VmBasedAgentHook] (AgentConnectTaskPool-4:[ctx-f64d74cc]) (logid:29169184) ConsoleProxySslEnabled.value() = true
```

however, I prefer not to change the code as the old/new codes basically get the same value from database. 
The old code has been proved to be working. Some other values are fetched in the same way.
@GutoVeronezi

Reply:
same as line-215